examples/nlp/masked_language_modeling.py

```diff
@@ -336,37 +376,48 @@ def metrics(self):
 
 
 def create_masked_language_bert_model():
```

Comment:
Here, shall I subclass the MaskedLanguageModel instead, and get rid of the tf custom training step?

Reply:
I believe you should be able to remove the `train_step()` and instead implement the `compute_loss()` method. If you write the `compute_loss()` method using only `keras.ops` functions, it will work with all backends.

Reply:
Given that no additional logic is required here, and using `SparseCategoricalCrossEntropy` as Loss function; does it require an override for `compute_loss()`?